packages/client/src/client.js

```diff
@@ -367,6 +368,7 @@ export class PercyClient {
     validateId('build', buildId);
     this.log.debug(`Uploading resources for ${buildId}...`, meta);
 
+    const uploadConcurrency = parseInt(process.env.PERCY_RESOURCE_UPLOAD_CONCURRENCY) || 2;
```

Comment:
is this new env var? Should it be added to readme/docs if new

Reply:
This we have added only for central scanner. i don't think some user needs to update this config.

Reply:
Ack